modules/oe_translation_cdt/modules/oe_translation_cdt_mock/tests/src/Unit/ServiceMockBaseTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
```

Comment:
I know that testing the mock may be too much, but I wanted to make sure that the utility functions are working fine.